doc/compodoc_sources/how-to-guides/add-cypress-to-project.md

```diff
@@ -0,0 +1,32 @@
+# How to add Cypress to project
```

Comment:
As no-one of the project will have to setup Cypress again I would remove this small guide and move the `How to use Cypress` to the other guide. To have it all in one place and don't clutter up the documentation with unnecessary guides.

Reply:
Done